pg/scripts/seed-data.sql

```diff
@@ -0,0 +1,11 @@
+-- ma-exec pg psql -f scripts/seed-data.sql -U postgres
+
+copy enzymes from '/input_data/enzymes/abac.txt' delimiter E'\t';
```

Comment:
Would it be possible to have something less hard-coded, eg '/input_data/enzymes/*.txt' ?